certification/runtime/config_test.go

```diff
@@ -0,0 +1,65 @@
+package runtime
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/spf13/viper"
+)
+
+var _ = Describe("Viper to Runtime Config", func() {
+	var baseViperCfg *viper.Viper
+	var expectedRuntimeCfg *Config
+	BeforeEach(func() {
+		baseViperCfg = viper.New()
```

Comment:
Doesn't have to be... But it feels like you could get away with a DescribeTable here? It's no longer an "extension" in ginkgo. It would get rid of the second context even. Feel free to ignore.

Reply:
I think the DescribeTable approach may require more edge case handling than it's wortyh because expectedRuntimeConfig is a struct without getters/setters to simplify it. With that said, I've added a struct key count check to make sure that Config changing causes a test failure, resulting in a change in tests with the included PR.